dist/cms.js

```diff
@@ -151,12 +151,31 @@ function getWordPressContentType() {
   return content;
 }
 
+/**
+ * Obtains data about Interactivity API elements on the page..
+ *
+ * @returns {object} Object with fields `usesInteractivityAPI`, `interactiveElementsCount`, and `interactiveElements`.
+ */
+function getInteractivityAPIData() {
+  // Look for data-wp-interactive elements.
+  const interactiveElements = document.querySelectorAll('[data-wp-interactive]');
+
+  return {
+    usesInteractivityAPI: interactiveElements.length > 0,
+    interactiveElementsCount: interactiveElements.length,
+    interactiveElements: Array.from(interactiveElements).map((element) => {
+      return element.dataset.wpInteractive;
+    } )
```

Comment:
This returns the data within `data-wp-interactive`, which is typically an object containing the namespace for the region, so calling it `interactiveElements` wouldn't be right.

I'm not even sure we want to provide every single interactive element (as that may end up a lot), or at least I feel like that would need to be thought through a bit more and could be something for a follow up.

I could see a value in having the interactive regions available including some of their data, such as namespaces. If we want to do that, how about calling this `interactiveRegions` and also providing some context of the element itself?

For example, instead of `return element.dataset.wpInteractive`, we could `return { tagName: element.tagName, wpInteractive: JSON.parse( element.dataset.wpInteractive ) }`? That allows us to expand what's included in the future if we come up with more useful things to include.

Reply:
> For example, instead of return element.dataset.wpInteractive, we could return { tagName: element.tagName, wpInteractive: JSON.parse( element.dataset.wpInteractive ) }? That allows us to expand what's included in the future if we come up with more useful things to include.

I'm going to try to build out a demo site with all of the available elements I can find that use the Interactivity API then explore further to see what the most useful data might be.  I agree we don't need to over-collect heremaybe just counts of each namespace? 

ps. I'm curious why your code used `JSON.parse( element.dataset.wpInteractive )` - is it expected this field can contain JSON data?

Reply:
> This returns the data within data-wp-interactive, which is typically an object containing the namespace for the region, so calling it interactiveElements wouldn't be right.

I've renamed the keys (and updated the PR description) so they are called regions. So far the namespace is a good reference point for the type of interactive element (and hopefully plugin authors will follow the core patterns). For example on my [test post](https://interactivity-api.instawp.xyz/test-post/), the regions have the namespaces:  `core/router`, `core/query`, `core/navigation`, `core/image` and `core/search`.